### PR TITLE
Update .travis.yml to test under PHP 8

### PR DESCRIPTION
**Pull request by sanmai**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ php:
   - 7.2
   - 7.3
   - 7.4
+  - 8.0snapshot
```
Comment (caugner) [inline .travis.yml:9]:
```suggestion
  - 8.0
```

**Comment by caugner** [comment]
FYI @sanmai
- [ ] `composer.json` needs to be updated to allow PHP 8.0, e.g. `>=5.3.3 <8.1`
- [ ] `friendsofphp/php-cs-fixer` needs to be updated to `v2.17.0`

**Review by caugner — COMMENTED** [review]
FYI @sanmai
- [ ] `composer.json` needs to be updated to allow PHP 8.0, e.g. `>=5.3.3 <8.1`
- [ ] `friendsofphp/php-cs-fixer` needs to be updated to `v2.17.0`

**Comment by sanmai** [comment]
Resumed the efforts in #31

**Pull request closed without merging**